internationalization/supportedLanguages.ts

```diff
@@ -0,0 +1,31 @@
+export interface Language {
+  _key: string;
+  title: string;
+  id: string;
+  icon: string;
+  default?: boolean;
+}
+
+export const supportedLanguages: Language[] = [
+  {
+    _key: `${Date.now()}-${Math.random().toString(36).slice(2, 11)}`,
```

Comment:
Any specific reason to use this id? Could we use UUID instead?

Reply:
nope just me forgetting to add the import 🙃